tasks/build-algolia-search.mjs

```diff
@@ -289,6 +289,21 @@ async function tryParseImports(
       source = source.join('\n');
     }
 
+    // remove INTEGRATION_FILTER_OPTIONS variables
```

Comment:
with the SSG update the pages meta data contains the `INTEGRATION_FILTER_OPTIONS` variable.  The library being used to extract the meta object from the mdx files `extract-mdx-metadata` is parsing out that meta object but crashes because `INTEGRATION_FILTER_OPTIONS` has not been defined.  Since this information isn't needed for the algolia indexing here we just remove those references entirely.